etc/job.yaml

```diff
@@ -48,45 +48,102 @@
+# settings for various other configs.
+#
+# The environment variable flight_JOB_scheduler takes precedence
+# ==============================================================================
+# scheduler: slurm
+
 # ==============================================================================
 # Submit Script Path
 # Specify the path to the script which submits jobs.
 #
-# Relative paths are expanded from the install directory.
+# The default value will automatic use the 'scheduler' config. This feature
+# is not supported when overridden.
+#
+# Relative paths are expanded from flight_ROOT. **
 #
 # The environment variable flight_JOB_submit_script_path takes precedence.
 # ==============================================================================
-# submit_script_path: libexec/slurm/monitor.sh
+# submit_script_path: libexec/<scheduler>/monitor.sh
 
 # ==============================================================================
 # Monitor Script Path
 # Specify the path to the script which monitors jobs.
 #
-# Relative paths are expanded from the install directory.
+# The default value will automatic use the 'scheduler' config. This feature
+# is not supported when overridden.
+#
+# Relative paths are expanded from flight_ROOT. **
 #
 # The environment variable flight_JOB_monitor_script_path takes precedence.
 # ==============================================================================
-# monitor_script_path: libexec/slurm/monitor.sh
+# monitor_script_path: libexec/<scheduler>/monitor.sh
+
+# ==============================================================================
+# Script Adapter Path
+# Specify the path to the script adapter/shim. This adapter will be rendered
+# between the 'directives' and 'workload' sections of the job's script. It is
+# used to apply FlightJob specific features.
+#
+# It is primarily used to set the RESULTS_DIR to match the value returned
+# by the 'submit script'. This directory will be scheduler specific.
+#
+# As this adapter is rendered into the middle of the script, it should not
+# contain the shebang (#!) line.
+#
+# The default value will automatic use the 'scheduler' config. This feature
+# is not supported when overridden.
+#
+# Relative paths are expanded from flight_ROOT. **
+#
+# The environment variable flight_JOB_adapter_script_path takes precedence.
+# ==============================================================================
+# adapter_script_path: usr/share/job/adapter.<scheduler>.erb
+
+# ==============================================================================
+# Template Map Path
+# Specify the path the template map path. This should contain scheduler specific
+# environment variables that will be rendered into the templates.
+#
+# This allows the templates to be written in a platform agnostic manner, but
+# rendered with familiar variables.
+#
+# The default value will automatic use the 'scheduler' config. This feature
```

Comment:
Copy change: automatic -> automatically.  And similar for the other directives please.